meta/AMuon/Analytic_Loop_Functions.m

```diff
@@ -0,0 +1,331 @@
+(* ::Package:: *)
+
+(* :Copyright:
+
+   ====================================================================
+   This file is part of FlexibleSUSY.
+
+   FlexibleSUSY is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published
+   by the Free Software Foundation, either version 3 of the License,
+   or (at your option) any later version.
+
+   FlexibleSUSY is distributed in the hope that it will be useful, but
+   WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+   General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with FlexibleSUSY.  If not, see
+   <http://www.gnu.org/licenses/>.
+   ====================================================================
+
+*)
+
+(*One Loop Functions*)
+OneLoopFunctionA[0]  := 2
+OneLoopFunctionA[1]  := 3/2.
```

Comment:
One note: can you please write `3/2` instead of `3/2.` here and in the other functions? The reason is that `3/2` can be evaluated with arbitrary precision, while `3/2.` is evaluated at (limited) machine precision.

Reply:
Sorry for nitpicking, but I think that special cases don't need `SetDelated`. So we can write
```mathematica
OneLoopFunctionA[1] = 3/2
```
i.e. `=` and not `:=`.

Reply:
All good, these suggestions for improving the code are very helpful.

Reply:
Great, thanks!